docs/minio-client-complete-guide.md

```diff
@@ -131,6 +135,19 @@ MinIO server displays URL, access and secret keys.
 mc config host add minio http://192.168.1.51 BKIKJAA5BMMU2RHO6IBB V7f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12 --api S3v4
 ```
 
+```sh
+mc config host add minio http://192.168.1.51
+```
+
+This prompts for details:
+
+```
+Enter Access Key : BKIKJAA5BMMU2RHO6IBB
+Enter Secret Key : V7f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12
+Enter API signature. An optional argument. Default value 'S3v4' :
+Enter lookup. An optional argument. Default value 'auto' :
```

Comment:
I believe both of these prompts are not there

Reply:
Thanks, removed.